app/controllers/users_controller.rb

```diff
@@ -5,6 +5,23 @@ def index
     user_addresses
   end
 
+  def show
+    find_user
```

Comment:
use a before_action :find_user

Reply:
actually, we already know who we are with `@current_user` or the `session`, I should not be able to see another user's profiles, so we don't need a show but we can add a profile action